web/app/setup.py

```diff
@@ -0,0 +1,83 @@
+import json
+import requests
+
+from flask import flash
+from flask import Flask
+from flask import redirect
+from flask import render_template
+from flask import request
+from flask import url_for
+from flask_bootstrap import Bootstrap
+from flask_wtf import Form
+from flask_wtf.csrf import CsrfProtect
+from wtforms import SubmitField
+from wtforms import TextField
+from wtforms.validators import ValidationError
+
+
+api_url = "http://localhost:5000/website"
```

Comment:
Can we move the actual urls to a config file/constants file? Then we can assign api_url = dev, prod, etc.